statsmodels/__init__.py

```diff
@@ -1,14 +1,14 @@
 from __future__ import print_function
+from .compat import PY3
 
-__docformat__ = 'restructuredtext'
+from warnings import simplefilter
 
-from distutils.version import LooseVersion
-import os
-import sys
+from ._test_runner import PytestTester
+from .tools.sm_exceptions import (ConvergenceWarning, CacheWriteWarning,
+                                  IterationLimitWarning, InvalidTestWarning)
+from ._version import get_versions
 
-from warnings import simplefilter
-from statsmodels.tools.sm_exceptions import (ConvergenceWarning, CacheWriteWarning,
-                                             IterationLimitWarning, InvalidTestWarning)
+__docformat__ = 'restructuredtext'
```

Comment:
Not for this PR, but are these lines still necessary?  They pop up in a bunch of places.

Reply:
Absolutely no idea.  I suspect not but don't need to fix everything now.